observability/k8s/datadog/values.yaml

```
@@ -0,0 +1,26 @@
+datadog:
+  profiling:
+    enabled: false
+  apiKeyExistingSecret: datadog-secret
+  site: datadoghq.com
+
+  containerExclude: "kube_namespace:.*"
+  containerInclude: "kube_namespace:bqnq kube_namespace:jonny kube_namespace:satomichi kube_namespace:taki"
```

このまま出しても問題はないのですが、わかりやすさのために個人名は抽象化しておきましょうか！
```suggestion
  containerInclude: "kube_namespace:member_ns1 "
```